src/System.IO.Ports/tests/SerialPort/BaudRate.cs

```diff
@@ -26,6 +26,7 @@ public class BaudRate_Property : PortsTest
         private enum ThrowAt { Set, Open };
 
         #region Test Cases
+        [KnownFailure] // either hanging or really slow
```

Comment:
What is the KnownFailure attribute?  How does that differ from ActiveIssue?

Reply:
this doesn't disable anything by default, it's more like a temporary thing which helps to manage test cases while I still work on them and gives hints what can still be broken and what should already work - I'd think of it as adding a xunit trait at this point. I can technically remove it since I don't believe any of those test cases run in the CI but I think it is still useful as long as still issues are being polished up

Reply:
note: those test cases were failing before (assuming you got null modem connected)